cypress/support/commands.js

```diff
@@ -0,0 +1,31 @@
+Cypress.Commands.add('login', user => {
```

Comment:
Excellent! :+1:
If we happen to use it a lot, we can create auxiliary methods `loginAsPatron`, `loginAsLibrarian`, etc.

Reply:
Yes, I agree, also it would be nice if we can hardcode some scenarios for some patrons, e.g. when we are testing the extension button we need to use a patron that will have a loan that can be extended.